packages/contract-artifacts/artifacts/ZRXToken.json

```diff
@@ -7,221 +7,150 @@
             },
             {
                 "constant": true,
-                "inputs": [
-                    {
-                        "name": "_owner",
-                        "type": "address"
-                    },
-                    {
-                        "name": "_spender",
-                        "type": "address"
-                    }
-                ],
+                "inputs": [{ "name": "_owner", "type": "address" }, { "name": "_spender", "type": "address" }],
                 "name": "allowance",
-                "outputs": [
-                    {
-                        "name": "",
-                        "type": "uint256"
-                    }
-                ],
+                "outputs": [{ "name": "", "type": "uint256" }],
                 "payable": false,
                 "type": "function"
             },
-            {
-                "inputs": [],
-                "payable": false,
-                "type": "constructor"
-            },
+            { "inputs": [], "payable": false, "type": "constructor" },
             {
                 "anonymous": false,
                 "inputs": [
-                    {
-                        "indexed": true,
-                        "name": "_from",
-                        "type": "address"
-                    },
-                    {
-                        "indexed": true,
-                        "name": "_to",
-                        "type": "address"
-                    },
-                    {
-                        "indexed": false,
-                        "name": "_value",
-                        "type": "uint256"
-                    }
+                    { "indexed": true, "name": "_from", "type": "address" },
+                    { "indexed": true, "name": "_to", "type": "address" },
+                    { "indexed": false, "name": "_value", "type": "uint256" }
                 ],
                 "name": "Transfer",
                 "type": "event"
             },
             {
                 "anonymous": false,
                 "inputs": [
-                    {
-                        "indexed": true,
-                        "name": "_owner",
-                        "type": "address"
-                    },
-                    {
-                        "indexed": true,
-                        "name": "_spender",
-                        "type": "address"
-                    },
-                    {
-                        "indexed": false,
-                        "name": "_value",
-                        "type": "uint256"
-                    }
+                    { "indexed": true, "name": "_owner", "type": "address" },
+                    { "indexed": true, "name": "_spender", "type": "address" },
+                    { "indexed": false, "name": "_value", "type": "uint256" }
                 ],
                 "name": "Approval",
                 "type": "event"
             }
         ],
+        "devdoc": {
```

Comment:
Hmm where did these devdocs come from? I don't think we want these in the artifacts

Reply:
They were auto-generated. Sounds like I should assume that anything not currently in there isn't wanted?

Reply:
Yes.